sdcm/nemesis.py

```diff
@@ -1815,6 +1844,16 @@ def disrupt(self):
         self.disrupt_toggle_table_ics()
 
 
+class MgmtBackup(Nemesis):
+    disruptive = False
+
+    @log_time_elapsed_and_status
+    def disrupt(self):
+        self.log.info('disrupt_mgmt_backup Nemesis begin')
```

Comment:
why did you put additional log output here? `log_time_elapsed_and_status` is doing this!